libpod/image/image.go

```diff
@@ -1570,6 +1570,16 @@ func (i *Image) GetHealthCheck(ctx context.Context) (*manifest.Schema2HealthConf
 	return configBlob.ContainerConfig.Healthcheck, nil
 }
 
+// newBuildEvent creates a new event based on completion of a built image
+func (ir *Runtime) NewImageBuildCompleteEvent(idOrName string) {
```

Comment:
Should we just put this in `libpod/runtime_img.go` as a method on Runtime instead of ImageRuntime? It's less consistent, but that way it stays in the same file as the only use of it...

Reply:
I waffled on that too.  In the end, I figured it should "sit' with the other methods.  I could go either way depending on how insistent we are.

Reply:
My basic thought it that we should make it private so nobody accidentally stumbles on the API and makes spurious image events

Reply:
Let's make it private, we can always make it public in the future if necessary without breaking API.